travis: [docutils 0.13.1] assert not self.context

In Travis, tests with minimal requirements are failing:

`Exception occurred:
  File "/home/plach/.pyenv/versions/3.5.2/envs/inv-dep-3.5.2/lib/python3.5/site-packages/docutils/writers/_html_base.py", line 671, in depart_document`
`assert not self.context, 'len(context) = %s' % len(self.context)`
`AssertionError: len(context) = 5`

It has already been raised in this [issue](https://github.com/sphinx-doc/sphinx/issues/3212).

It can be fixed by upgrading Shinx to 1.5.1 in `setup.py`.

--- a/setup.py
+++ b/setup.py
@@ -49,7 +49,7 @@ tests_require = [
 
 extras_require = {
     'docs': [
-        'Sphinx>=1.4.2',
+        'Sphinx>=1.5.1',
     ],
     'tests': tests_require,
 }

